ui/app/AppLayouts/Wallet/panels/BuyCryptoProvidersListPanel.qml

```diff
@@ -0,0 +1,77 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.0
+import QtQml.Models 2.14
+import SortFilterProxyModel 0.2
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+
+import AppLayouts.Wallet.controls 1.0
+
+ColumnLayout {
+    id: root
+
+    // required properties
+    required property bool providersLoading
+    required property var providersModel
+    required property bool isUrlBeingFetched
+    required property string selectedProviderId
+
+    // exposed api
+    property alias currentTabIndex: tabBar.currentIndex
+    signal providerSelected(var model)
```

Comment:
This should ideally emit just the needed parameter (some provider ID?), not the whole model

Reply:
need two params `id` and `urlsNeedParameters` can still make them as params in the signal, but dont see any big benifit?

Reply:
Well, size (2 versus many params) and readability (explicit names and types)

Reply:
Yeah, I agree with @caybro. Detailed rationale is provided in 5th point of https://github.com/status-im/status-desktop/blob/7d59d606dbef6c726d14efc9dc2dac2985930410/architecture-guide.md#general-architecture-good-practices The emitting component shouldn't care what is used externally. And changing external usage shouldn't trigger changes in this component.